.gitignore

```diff
@@ -49,6 +49,7 @@ modules.order
 Module.symvers
 Mkfile.old
 dkms.conf
+dist/
```

Comment:
what is this now excluding? The hex files?

Reply:
Got created when I first opened the project with the latest MPLAB. It might be something left over from the conversion to the newer MPLAB.

Reply:
can we remove it for now as an uneeded change?

Reply:
That's odd - I just tried it again and it didn't create it. Maybe I imagined it.
Will remove.